ui/apps/platform/cypress/helpers/systemHealth.js

```diff
@@ -58,11 +59,17 @@ export function visitSystemHealthFromLeftNav() {
     visitFromLeftNavExpandable('Platform Configuration', 'System Health', routeMatcherMap);
 
     cy.location('pathname').should('eq', systemHealthUrl);
-    cy.get('h1:contains("System Health")');
+    cy.get(SystemHealthHeadingSelector);
 }
 
 export function visitSystemHealth(staticResponseMap) {
     visit(systemHealthUrl, routeMatcherMap, staticResponseMap);
 
-    cy.get('h1:contains("System Health")');
+    cy.get(SystemHealthHeadingSelector);
+}
+
+export function visitSystemHealthWithStaticResponseForCapabilities(staticResponseForCapabilities) {
+    visitWithStaticResponseForCapabilities(systemHealthUrl, staticResponseForCapabilities);
```

Comment:
Include `routeMatcherMap` as third argument of function call, to wait for page-specific requests.